vt/client.py

```diff
@@ -96,7 +96,9 @@ def read(self):
     return _make_sync(self.read_async())
 
   async def json_async(self):
-    return await self._aiohttp_resp.json()
+    content_length = int(self.headers.get('content-length', '-1'))
+    response_content = await self.content.read_async(content_length)
+    return json.loads(response_content)
```

Comment:
Assuming this fix works, I think the `read_async` method of the `ClientResponse` class (line 92) should also be updated to get the `content length` .

Reply:
That was the first thing I tried, but only reads from [`self.content` (i.e. a `StreamReader` object)](https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse.content) have the possibility of specifying how much data to read. The ClientResponse's [`read`](https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse.read) or [`json`](https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse.json) methods don't have that option.

Reply:
Right, so couldn't you just replace line 96 with:
```python
content_length = int(self.headers.get('content-length', '-1'))
return await self.content.read_async(content_length)
```

Reply:
OBE, see #56